packages/netsuite-adapter/src/adapter.ts

```
@@ -257,44 +257,43 @@ export default class NetsuiteAdapter implements AdapterOperations {
     const updatedFetchQuery = changedObjectsQuery !== undefined
       ? andQuery(changedObjectsQuery, fetchQuery)
       : fetchQuery
+    const queries = { updatedFetchQuery, originFetchQuery: fetchQuery }
```

I like this PR 🙏 
One main comment is about `getConfigRecords`

I think it should fail fast on (invalid creds), and that does not happen today
See my comment here - https://github.com/salto-io/salto/pull/5009#discussion_r1371843042

Maybe we should address this here as well?
thoughts?

---

will let @omrilit approve this ofc. but from my point of view. LGTM 👍

---

@dantal4 do you remember why we return an empty array when failing to fetch the config records? is it because it may happen consistently to certain customers?

---

We saw accounts that weren't able to fetch specific config types, but that is a different case. I think that we just never gave a deep thought on it.